### PR TITLE
Update LICENSE

### PR DESCRIPTION
**Pull request by lhstrh**

### CODE REVIEW

**Comment by lhstrh** [comment]
This is rendered as follows:

image

File: LICENSE.md [hunk 1]
```diff
@@ -1,25 +1,30 @@
-The Lingua-Franca toolkit is licensed under the BSD 2-Clause License:
+BSD 2-Clause License
 
-Copyright (c) 2019-2020, Industrial Cyberphysical Systems (iCyPhy). All rights reserved.
+Copyright (c) 2019-present, The Lingua Franca authors:
 
-Redistribution and use in source and binary forms, with or without modification, 
-are permitted provided that the following conditions are met:
+<a href="https://github.com/lf-lang/lingua-franca/graphs/contributors">
```
Comment (lhstrh) [inline LICENSE.md:5]: If we want to stay in control of these links, we should create aliases for them, like:
```suggestion
<a href="https://contrib.lf-lang.org">
```

File: LICENSE.md [hunk 2]
```diff
@@ -1,25 +1,30 @@
...
+  <img src="https://contrib.rocks/image?repo=lf-lang/lingua-franca" />
```
Comment (lhstrh) [inline LICENSE.md:6]:
```suggestion
  <img src="https://authors.lf-lang.org" />
```

File: LICENSE.md [hunk 3]
```diff
@@ -1,25 +1,30 @@
-The Lingua-Franca toolkit is licensed under the BSD 2-Clause License:
+BSD 2-Clause License
```
Comment (lhstrh) [inline LICENSE.md:1]:
```suggestion
This software is licensed under the BSD 2-Clause License
```

File: LICENSE.md [hunk 4]
```diff
@@ -1,25 +1,30 @@
...
 
-Copyright (c) 2019-2020, Industrial Cyberphysical Systems (iCyPhy). All rights reserved.
+Copyright (c) 2019-present, The Lingua Franca authors:
 
-Redistribution and use in source and binary forms, with or without modification, 
-are permitted provided that the following conditions are met:
+<a href="https://contrib.lf-lang.org">
+  <img src="https://authors.lf-lang.org" />
```
Comment (lhstrh) [inline LICENSE.md:6]:
```suggestion
  <img src="https://contrib-img.lf-lang.org" />
```

File: LICENSE.md [hunk 5]
```diff
@@ -1,25 +1,30 @@
-The Lingua-Franca toolkit is licensed under the BSD 2-Clause License:
+BSD 2-Clause License
 
-Copyright (c) 2019-2020, Industrial Cyberphysical Systems (iCyPhy). All rights reserved.
+Copyright (c) 2019-present, The Lingua Franca authors:
```
Comment (lhstrh) [inline LICENSE.md:3]:
```suggestion
Copyright (c) 2019-present, The Lingua Franca contributors:
```

File: LICENSE.md [hunk 6]
```diff
@@ -1,25 +1,30 @@
-The Lingua-Franca toolkit is licensed under the BSD 2-Clause License:
+This software is licensed under the BSD 2-Clause License
```
Comment (lhstrh) [inline LICENSE.md:1]:
```suggestion
This software is licensed under the BSD 2-Clause License.
```

File: LICENSE.md [hunk 7]
```diff
@@ -1,25 +1,30 @@
...
 
-Copyright (c) 2019-2020, Industrial Cyberphysical Systems (iCyPhy). All rights reserved.
+Copyright (c) 2019-present, The Lingua Franca authors:
```
Comment (lhstrh) [inline LICENSE.md]:
```suggestion
Copyright (c) 2019-present, The Lingua Franca authors. All rights reserved.
```

File: LICENSE.md [hunk 8]
```diff
@@ -1,25 +1,30 @@
...
 
-Redistribution and use in source and binary forms, with or without modification, 
-are permitted provided that the following conditions are met:
+<a href="https://contrib.lf-lang.org">
+  <img src="https://contrib-img.lf-lang.org" />
+</a>
 
-Redistributions of source code must retain the above copyright notice, 
-this list of conditions and the following disclaimer.
+All rights reserved.
```
Comment (lhstrh) [inline LICENSE.md:9]:
```suggestion
```

File: LICENSE.md [hunk 9]
```diff
@@ -1,25 +1,30 @@
-The Lingua-Franca toolkit is licensed under the BSD 2-Clause License:
+This software is licensed under the BSD 2-Clause License
 
-Copyright (c) 2019-2020, Industrial Cyberphysical Systems (iCyPhy). All rights reserved.
+Copyright (c) 2019-present, The Lingua Franca contributors:
```
Comment (lhstrh) [inline LICENSE.md:3]:
```suggestion
Copyright (c) 2019-present, The Lingua Franca contributors. All rights reserved.
```

**Comment by edwardalee** [comment]
I think a copyright notice cannot meaningfully be "2019-present". Since copyrights expire, the date cannot be later than the last date of authorship.  Unfortunately, we need a specific year, which means needing a script to update the date.

**Comment by lhstrh** [comment]
> I think a copyright notice cannot meaningfully be "2019-present". Since copyrights expire, the date cannot be later than the last date of authorship. Unfortunately, we need a specific year, which means needing a script to update the date.

I don't think that it's necessary to specify an end date though. Should we just leave it out? I'm also fine incorporating the copyright year update in the release workflow.

**Comment by edwardalee** [comment]
I think that if we don't specify an end date, then we will perpetually be claiming it was all written in 2019.

**Comment by lhstrh** [comment]
AFAIK Copyright claims are not at all contingent on have declared copyright in any shape or form, so I don't think it matters much. The resources I reached via Google all say it's not necessary. That said, I'm totally fine with adding an end date and updating it as part of our release cycle. Shall we settle on that then?

File: LICENSE.md [hunk 10]
```diff
@@ -1,25 +1,28 @@
-The Lingua-Franca toolkit is licensed under the BSD 2-Clause License:
+This software is licensed under the BSD 2-Clause License.
 
-Copyright (c) 2019-2020, Industrial Cyberphysical Systems (iCyPhy). All rights reserved.
+Copyright (c) 2019-present, The Lingua Franca contributors. All rights reserved.
```
Comment (lhstrh) [inline LICENSE.md:3]:
```suggestion
Copyright © 2019-2022, The Lingua Franca contributors. All rights reserved.
```

Comment (lhstrh) [reply LICENSE.md:3]: @edwardalee do you think the capitalization here checks out?

**Comment by edwardalee** [comment]
My concern is not the legality of the statement but the optics. It will very quickly make everything look old an inactive.

File: LICENSE.md (hunk 10 above)
Comment (edwardalee) [reply LICENSE.md:3]: Looks good to me.  Is the non-ASCII character going to bite (byte?) us?

**Review by cmnrd — COMMENTED** [review]
This looks good to me. I am not sure if we should include images of all the contributors in a license, though. I think this would be more fitting for a contributing page, or a dedicated page that introduces the team.

**Pull request merged**